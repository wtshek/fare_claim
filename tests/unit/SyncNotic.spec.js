import { shallowMount } from '@vue/test-utils'
import SyncNotification from '../../components/notificationBox'

describe('Sync Notifications', () => {
  it('renders the component', () => {
    const wrapper = shallowMount(SyncNotification, {
      propsData: {
        message: 'Syncing Data',
        show: false,
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
    expect(wrapper.props().message).toBe('Syncing Data')
    expect(wrapper.props().show).toBe(false)
  })
})
