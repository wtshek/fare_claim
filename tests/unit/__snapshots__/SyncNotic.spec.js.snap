// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sync Notifications renders the component 1`] = `
<div class="not-show container">
  <div class="message">Syncing Data</div>
  <b-progress type="is-primary" class="progress-bar"></b-progress>
</div>
`;
